tony-core/src/main/java/com/linkedin/tony/runtime/HorovodRuntime.java

```diff
@@ -51,6 +51,9 @@
 public class HorovodRuntime extends MLGenericRuntime {
     private static final String DRIVER = "driver";
     private static final String WORKER = "worker";
+    private static final List<String> ILLEGAL_CONFIG_KEYS = Arrays.asList(
+            "tony.driver.instances"
+    );
```

Comment:
This is very hard to understand... I think what you want to prevent is, you don't want users to have more than 1 driver?

Reply:
Yes. Actually user shouldn't configure config about driver role.
Currently because we will overwrite the user configuration, the job will not throw an exception. But this behavior is actually contrary to user expectations, so I think fast fail is more reasonable.

Reply:
Right, then it should be `tony.driver.*`? would regex matching makes more sense or over engineering..? like ["!tony.driver.*"]

Reply:
Sounds great!